pyproject.toml

```diff
@@ -70,3 +72,7 @@ exclude = [
   "mlflow/temporary_db_migrations_for_pre_1_users",
   "tests/protos",
 ]
+[tool.ruff.per-file-ignores]
+"dev/*" = ["T201", "T203"]
```

Comment:
do we need T203?

Reply:
No, I will delete it. Thought to add because task was to add T20 )